### DISCUSSION TITLE
Orphaned .zip files

### DISCUSSION THREAD

**Discussion by gvanem** [microsoft/vcpkg#19414] [Q&A]
I just discovered there are 3 GByte of .zip files under `c:\Users\Gisle\AppData\Local\vcpkg\archives\`
that seems not to belong to any installed package. 
E.g. in `<VCPKG_ROOT>/installed/vcpkg/status` file there is:

```
Package: icu
Version: 69.1
...
Abi: 75d25bd26db1a3e848e18ccdb53bb9053772d270
...
Status: purge ok not-installed
```
The above `Abi:` matches a `c:\Users\Gisle\AppData\Local\vcpkg\archives\75\75d25bd26db1a3e848e18ccdb53bb9053772d270.zip` file.
And many more similar cases.

What command do I use to clean up this disk-waste? Or can I simply remove all?

**Comment by Neumann-A** [comment]
> c:\Users\Gisle\AppData\Local\vcpkg\archives\

That is the default binary caching location. It can deactivated be setting `VCPKG_DEFAULT_BINARY_CACHE` to empty. https://github.com/microsoft/vcpkg/blob/master/docs/users/binarycaching.md
 
 > What command do I use to clean up this disk-waste? Or can I simply remove all?
 
there is no command for cleanup. You can remove all but this means that other vcpkg instances on the same PC have to rebuild the ports instead of taking them from the cache

**Reply by gvanem** [reply]
> You can remove all but this means that ...

Good. Thanks!

Another question. These 2 files seems to have been signed with 2 different hash-algorithms:
```
  c:\Users\Gisle\AppData\Local\vcpkg\archives\73\73466ade06156084747499fbb097185d1e861336.zip
  c:\Users\Gisle\AppData\Local\vcpkg\archives\74\748e912d5dbe805d05e44b7350db26ab1188f157b724ce30ded81bef82f5fdd3.zip
```

(different length). Which one are those?
When trying all the Cygwin programs `shaXXsum.exe` on these, I get no match on these filenames for their content.
Or are the fingerprint based on something else than their contents?
